Sources/MeiliSearch/Model/KeyParams.swift

```diff
@@ -67,3 +83,9 @@ public struct KeyUpdateParams: Codable, Equatable {
     }
   }
 }
+
+public extension KeyParams {
+  var enumActions: [KeyAction] {
+    self.actions.map { KeyAction($0) }
+  }
+}
```

Comment:
What are the thoughts on making this a computed variable as opposed to simply changing `actions` from a [String] to [KeyAction].

It's a breaking change, but I feel best to store as "typed" as possible.

Reply:
I decided to do it that way, so it’d be an addition but not required. If you prefer it as replacement, I’m happy to change it

Reply:
I think given we're still getting the API in a solid place, I'd want a core principle of this library to be type safety -- and thus would encourage storing it as type safe solution as opposed to the string array.

This is a discussion not a decision though, so if you disagree let's chat about that.

Reply:
It's less of a concern and more an awareness that we're storing it in its most compressed state (a String) which delays the point at which it becomes typed.

It also requires users to know that the extension exists, where default behaviour will be to use the string variable.

I think by replacing the array type we ensure users are using the optimal approach by default, and have to go out of their way to use strings.

Reply:
I agree on that. I'm going to implement it later or in the next few days :)